--- ui/tests/CMakeLists.txt
+++ ui/tests/CMakeLists.txt
@@ -21,7 +21,6 @@
         if (TARGET Qt6::${qt_comp})
             get_target_property(comp_libs Qt6::${qt_comp} INTERFACE_LINK_LIBRARIES)
             if(comp_libs)
-                string(REPLACE "-framework;AGL" "" comp_libs "${comp_libs}")
                 string(REPLACE "-framework AGL" "" comp_libs_new "${comp_libs}")
                 string(REPLACE "-framework;AGL" "" comp_libs_new "${comp_libs_new}")
                 set_target_properties(Qt6::${qt_comp} PROPERTIES INTERFACE_LINK_LIBRARIES "${comp_libs_new}")
